mirrord/kube/src/api/runtime.rs

```diff
@@ -101,6 +107,52 @@ impl RuntimeData {
             container_name,
         })
     }
+
+    pub async fn check_node(&self, client: &kube::Client) -> Result<()> {
```

Comment:
going with my idea that this should be optional (can fail due to permissions and would be ok) maybe make it `Result<bool>` for whether check was even done?

Reply:
Updated with a custom type